hackathon_site/dashboard/frontend/src/pages/Dashboard/Dashboard.js

```diff
@@ -37,6 +89,11 @@ const Dashboard = () => {
                 ))}
                 <ConnectedSponsorCard />
             </Grid>
+            {orderStatus !== null && (
+                <ItemTable title="Order pending" items={itemsP} status={orderStatus} />
+            )}
+            <ItemTable title="Checked out items" items={itemsC} status={orderStatus} />
+            <ItemTable title="Returned items" items={itemsR} status={orderStatus} />
```

Comment:
A simple test to check that these are on the page would be helpful (use a `getByText` on the table title or something).